components/home/__tests__/homepage-whatsapp-cta.test.tsx

```diff
@@ -44,4 +45,12 @@ describe("HomePageContributing", () => {
       </section>
     `);
   });
+
+  it("should navigate to ", () => {
```

Comment:
Please complete this sentence.
```suggestion
  it("navigates to the configured WhatsApp contact URL", () => {
```

Reply:
updated with the requested change @zainfathoni